recipe/build.sh

```diff
@@ -57,6 +58,16 @@ if [[ "$target_platform" == "linux-aarch64" ]]; then
   TEST_CMD=""
 fi
 
+if [[ "$target_platform" == "osx-arm64" ]]; then
+  ARCH_OPTS_SINGLE="--enable-neon"
+  ARCH_OPTS_DOUBLE="--enable-neon"
+  ARCH_OPTS_LONG_DOUBLE=""
```

Comment:
This wouldn't disable the long double library, this would override the double library which we do not want